sympy/functions/elementary/trigonometric.py

```diff
@@ -1745,6 +1761,9 @@ def fdiff(self, argindex=1):
         else:
             raise ArgumentIndexError(self, argindex)
 
+    def _eval_rewrite_as_besselj(self, arg, **kwargs):
+        from sympy.functions.special.bessel import besselj
+        return Piecewise((1/(sqrt(2)*sqrt(pi)*sqrt(arg)*besselj(-S.Half, arg)/2), Ne(arg, 0)), (1, True))
```

Comment:
The `/2` and the `sqrt(2)` partially cancel so we can just have `sqrt(2)` in the numerator.